app/main/view.ts

```diff
@@ -17,7 +17,7 @@ export class View extends BrowserView {
 	index: number;
 	url: string;
 	zoomFactor: number;
-	loading: true;
```

Comment:
Wow, I am surprised typescript didn't complain here.

Reply:
Removed loading completely from here and moved it to main.ts